File: src/client/xnative/lion.xdroid/Control/StorePagerFragment.cs
using Android.Content.PM;
using Android.OS;
using Android.Support.V4.App;
using Android.Support.V4.Content;
using Android.Views;
using Android.Webkit;
using Lion.XDroid;

namespace Lion.XDroid.Control
{
    public class StorePagerFragment : Android.Support.V4.App.Fragment, IPagerFragment
    {
        View rootView;

        public class StoreWebChromeClient : WebChromeClient
        {
            public override void OnGeolocationPermissionsShowPrompt(string origin, GeolocationPermissions.ICallback callback)
            {
                callback.Invoke(origin, true, false);
            }
        }

        public StorePagerFragment()
        {
        }

        public override void OnCreate(Bundle savedInstanceState)
        {
            //Log.Debug(this.GetType().Name, $"OnCreate");
            base.OnCreate(savedInstanceState);

            var _permission_check = ContextCompat.CheckSelfPermission(this.Context, Android.Manifest.Permission.AccessFineLocation);
            if (_permission_check == Permission.Denied)
                ActivityCompat.RequestPermissions(this.Activity, new string[] { Android.Manifest.Permission.AccessFineLocation }, 1);
        }

        public override View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
        {
            //Log.Debug(this.GetType().Name, $"OnCreateView ({((ViewPager)container).CurrentItem})");

            this.rootView = inflater.Inflate(Resource.Layout.fragment_store, container, false);
            {
                var _webview = this.rootView.FindViewById<WebView>(Resource.Id.wv_store);
                {
                    _webview.SetWebViewClient(new WebViewClient());
                    _webview.Settings.JavaScriptEnabled = true;
                    _webview.Settings.SetGeolocationEnabled(true);
                    _webview.SetWebChromeClient(new StoreWebChromeClient());

                    _webview.LoadUrl("https://m.map.daum.net/actions/searchView?q=로또%20판매점&tab=place&service=dksearch&viewmap=true");
                }
            }

            return this.rootView;
        }

        public void Initialize()
        {
        }
    }
}

File: src/client/xcode/Lion.Droid/app/src/main/java/com/odinsoftware/lottolion/control/StorePagerFragment.java
package com.odinsoftware.lottolion.control;

import android.Manifest;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.support.v4.app.ActivityCompat;
import android.support.v4.app.Fragment;
import android.support.v4.content.ContextCompat;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.GeolocationPermissions;
import android.webkit.WebChromeClient;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import com.odinsoftware.lottolion.R;

/**
 * Created by limyg on 2017-04-26.
 */
public class StorePagerFragment extends Fragment {

    View rootView;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        rootView = inflater.inflate(R.layout.fragment_store, container, false);
        WebView webView = (WebView)rootView.findViewById(R.id.wv_store);
        webView.setWebViewClient(new WebViewClient());
        webView.getSettings().setJavaScriptEnabled(true);
        webView.getSettings().setGeolocationEnabled(true);
        webView.setWebChromeClient(new WebChromeClient() {
            public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermissions.Callback callback) {
                callback.invoke(origin, true, false);
            }
        });
        int permissionCheck = ContextCompat.checkSelfPermission(getContext(), Manifest.permission.ACCESS_FINE_LOCATION);

        if(permissionCheck== PackageManager.PERMISSION_DENIED){
            ActivityCompat.requestPermissions(getActivity(), new String[]{Manifest.permission.ACCESS_FINE_LOCATION},1);
        }
        webView.loadUrl("https://m.map.daum.net/actions/searchView?q=로또%20판매점&tab=place&service=dksearch&viewmap=true");

        return rootView;
    }
}
